sites/bizbash/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = startServer({
   redirectHandler: ({ from }) => {
     const pattern = /(\/story\/[0-9]*|\/listing\/[0-9]*|\/gallery\/[0-9]*)/;
     const matches = pattern.exec(from);
-    if (!matches || !matches[0]) return null;
-    return { to: matches[0] };
+    if ((matches || matches[0]) && matches[0] !== matches['input']) return { to: matches[0] };
+    return null;
```

Comment:
I would clean this up into three conditional steps, instead of combining ORs and ANDs